src/v2/Components/NavBar/menuData.ts

```diff
@@ -270,8 +270,8 @@ export const ARTWORKS_SUBMENU_DATA: MenuLinkData = {
               href: "/collection/sculpture",
             },
             {
-              text: "Drawing",
-              href: "/collection/drawing",
+              text: "Work on Paper",
```

Comment:
Hmmm this is a tough one...I think it should also be `Works on Paper`. Let's make that change, and I'll confirm with the team responsible